pelican/tests/test_log.py

```diff
@@ -80,3 +83,61 @@ def do_logging():
             self.assertEqual(
                 self.handler.count_logs('Another log \\d', logging.WARNING),
                 0)
+
+    def test_fatal_logger_no_quit(self):
+        # log.FatalLogger.fatal_log_level = logging.NOTSET
+        log.init(logging.DEBUG, fatal='')
+
+        for level in ('debug', 'info', 'warning', 'error', 'critical'):
+            self._log_action(level)('%s: FatalLogger.%s()' % (__name__, level))
```

Comment:
I don't have a strong aversion to the `_log_action` method but just to give an alternative, loggers have `log` generic method as well. With that you can write this as:

```python
for level in (logging.DEBUG, logging.INFO, logging.WARNING, 
              logging.ERROR, logging. CRITICAL):
    self.logger.log(level, '%s: FatalLogger.%s()', __name__, level)
```

PS: Also note that, I changed the `"..." % (args, ...)` to `"...", args, ...`. It doesn't matter much but it will be [consistent with the rest of pelican](https://docs.getpelican.com/en/latest/contribute.html#logging-tips) :).

Reply:
Yes I like that much better.  Don't want to reinvent the wheel here!
And thanks for the link!